src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx

```diff
@@ -201,37 +226,25 @@ The **Map view** button is only available for the following types of entities:
   </tbody>
 </table>
 
-## Explore the history
-
-Unlike simple navigation maps, such as New Relic [maps in context](/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context/), automap helps you understand how problems developed by allowing you to scroll backwards and forwards in time. You can also screen out entities you're not interested in.
-
-### Timewarp cursor: go back in time [#timewarp-cursor]
-
-To identify exactly when and where an incident originated, simply move the **Timewarp** cursor to see health status changes for the entities on the map. If you don't see the **Timewarp** cursor, click the clock icon in the upper-right of the automap. When you scroll through the history, you see when anomalous behavior was detected by New Relic Lookout.
+## Timewarp cursor: Go back in time [#timewarp-cursor]
 
-Here’s an example of scrolling that reveals that the issue occurred between `FulfillmentService` and `BillingService`:
-
-<Video
-  type="wistia"
-  id="xqj83ghoog"
-/>
-
-### Controls: See only what you need to see [#filter-automap]
-
-With the **controls** menu available below the **Timewarp** cursor, you can de-emphasize certain entities on the map by unchecking those options. For example, you can hide the healthy service to focus only on those alerting as you browse the timeline:
+Once you've settled on your desired relationship depth, you can use our timewarp feature to explore changes between these entities from the last three hours. Simply move the **Timewarp** cursor to see health status changes for the entities on the map. 
 
 <img
-  title="Screenshot showing where to uncheck options"
-  alt="Screenshot showing where to uncheck options"
-  src={layerControlCheckBoxes}
+  title="Timewarp in automap view"
+  alt="A screenshot of where to find the timewarp feature in the automap UI. You can find timewarp by going to the top pane and moving to the right."
+  src={omnimapTimewarp}
 />
 
-Here is an example of how the controls work:
+## Controls: See only what you need to see [#filter-automap]
 
-<Video
-  type="wistia"
-  id="e15c99hsr2"
-/>
+With the **Health status** and **Entity types** dropdowns available to the left of **Timewarp**, you can de-emphasize certain entities on the map by unchecking those options. For example, you can hide the healthy service to focus only on those alerting as you browse the timeline:
```

Comment:
Health status dropdown only hides the entity health outline around the node(s) -- de-emphasizing them. It does not hide any nodes.

Entity type dropdown _will_ hide entities of the type(s) that are unchecked.